plugins/darktrace/icon_darktrace/util/api.py

```diff
@@ -50,8 +56,49 @@ def _get_signature(self, request: str, date: str) -> str:
             hashlib.sha1,
         ).hexdigest()
 
-    def _call_api(self, method: str, path: str, data: dict = None, params: dict = None) -> dict:
+    def _get_headers(self, endpoint: str) -> Dict[str, Any]:
+        """Returns the headers needed for all the Darktrace API requests
+
+        :param endpoint: Endpoint for headers to be generated
+        :type endpoint: str
+
+        :return: Dictionary contains headers needed for Darktrace API requests
+        :rtype: Dict[str, Any]
+        """
+
         date = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        headers = {
+            "DTAPI-Token": self.public_token,
+            "DTAPI-Date": date,
+            "DTAPI-Signature": self._get_signature(endpoint, date),
+        }
+        return headers
+
+    def _process_request_response(self, response: requests.Response) -> Dict[str, Any]:
+        """Returns the headers needed for all the Darktrace API requests
```

Comment:
This line is a copy of the _get_headers doc string.

Reply:
Oh, yea. You're right! My bad 😄